src/api/hooks/useTreasuryInfo.ts

```diff
@@ -20,6 +23,25 @@ export function useTreasuryInfo() {
 
     const accountInfos = await getAccountsIdentityInfo(api, accountIds);
 
-    return {proposals: proposals, accountInfos: accountInfos};
+    const treasuryAccount = u8aConcat(
+      'modl',
+      api.consts.treasury && api.consts.treasury.palletId ? api.consts.treasury.palletId.toU8a(true) : 'py/trsry',
+      EMPTY_U8A_32,
+    ).subarray(0, 32) as AccountId;
+
+    const treasuryBalance = await api.derive.balances.account(treasuryAccount);
+
+    const burn =
+      treasuryBalance?.freeBalance.gtn(0) && !api?.consts.treasury.burn.isZero()
```

Comment:
The `api` will always be defined at this point
```suggestion
      treasuryBalance?.freeBalance.gtn(0) && !api.consts.treasury.burn.isZero()
```

The same for other places where we do `api?.` here

Reply:
done ✅